homeassistant/components/flux_led/config_flow.py

```diff
@@ -0,0 +1,145 @@
+"""Config flow for Flux LED/MagicLight."""
+import logging
+
+from flux_led import BulbScanner, WifiLedBulb
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TYPE
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for FluxLED/MagicHome Integration."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    async def async_step_import(self, import_config: dict = None):
+        """Handle configuration via YAML import."""
+        _LOGGER.info("Importing configuration from YAML for flux_led.")
```

Comment:
Debug max. We don't end logging messages with period.

Reply:
No problem - fixed.

Reply:
Missed this one before... not sure why. Fixed now.